src/libosd/Makefile.am

```diff
@@ -10,6 +10,7 @@ pkginclude_HEADERS =\
 	include/osd/cl_scm.h \
 	include/osd/cl_stm.h \
 	include/osd/cl_ctm.h \
+        include/osd/cl_cdm.h \
```

Comment:
Please fix the indentation here to be consistent (you need to use tabs in Makefiles).